src/nupic/algorithms/Cells4.cpp

```diff
@@ -1336,37 +1326,7 @@ void Cells4::compute(Real* input, Real* output, bool doInference, bool doLearnin
   // invalidating our indexes.
   memset(output, 0, _nCells * sizeof(output[0])); // most output is zero
 #if SOME_STATES_NOT_INDEXED
-#ifdef NTA_PLATFORM_darwin86
```

Comment:
I'm not familiar with the history of these optimizations. What impact does this have? Is it a speed-only change or does it impact the actual results? And how much of a speed impact does it have?